### PR TITLE
Simplified the JavaScript

### PR DESCRIPTION
**Pull request by ThomasHickman**
Simplified the JavaScript

### CODE REVIEW

File: Javascript/jq-test.js [hunk 1]
```diff
@@ -1,69 +1,17 @@
-$( document ).ready(function() {
-    console.log( "ready!" );
```
Comment (ThomasHickman) [inline Javascript/jq-test.js:2]: Just getting rid of the logging

Comment (ThomasHickman) [reply Javascript/jq-test.js:2]: `$( document ).ready(function() {` is the same as `$(function() {`, so nesting it does nothing, I've just added one top level `$(function(){ ... }` which does the same thing

File: Javascript/jq-test.js [hunk 2]
```diff
@@ -1,69 +1,17 @@
...
-
+$(function() {
     $('#settings').click(function(){
       $('.bkg-menu').removeClass("hide");
```
Comment (ThomasHickman) [inline Javascript/jq-test.js:3]: I've combined the individual `$('#settings').click(function() {` blocks into one, as this produces the same effect

File: Javascript/jq-test.js [hunk 3]
```diff
@@ -1,69 +1,17 @@
...
-    });
-
-    $(function() {
-
-        $('#settings').click(function() {
-
-            if ($('#set-title').hasClass("hide")) {
-                $('#set-title').removeClass('hide');
-            }
-            else if (!$('#set-title').hasClass("hide")) {
```
Comment (ThomasHickman) [inline Javascript/jq-test.js:23]: `#settings` cannot be clicked when `#set-title` is visible, so this is and the if are redundant. Also `else if (!$('#set-title').hasClass("hide")) {` would be the same as `else {`, if this was needed, as if `$('#set-title').hasClass("hide")` is false, `!$('#set-title').hasClass("hide")` is always true

File: Javascript/jq-test.js [hunk 4]
```diff
@@ -1,69 +1,17 @@
...
-        $('#menu-close').click(function() {
-
-            if (!$('#pro-title').hasClass("hide")) {
-                $('#pro-title').addClass('hide');
-            }
-
-        });
-
+      $('#set-title').addClass('hide');
+      $('#pro-title').addClass('hide');
```
Comment (ThomasHickman) [inline Javascript/jq-test.js:15]: Not added an if statement here as if `#pro-title` already has the class `hide`, `$('#pro-title').addClass('hide');` will do nothing anyway

**Comment by bashpro** [comment]
thanks

**Pull request closed without merging**